chatmaild/src/chatmaild/filtermail.py

```diff
@@ -104,8 +105,8 @@ def check_armored_payload(payload: str, outgoing: bool):
     # Disallow comments in outgoing messages
     version_comment = "Version: "
     if payload.startswith(version_comment):
-        version_line = payload.splitlines()[0]
-        payload = payload.removeprefix(version_line)
+        splitindex = payload.find("\r\n") + 4
```

Comment:
Why even do `find` for outgoing messages if the result is not used?

Reply:
`find()` returns the first occurence of a line break as int. It is used one line later to remove the version comment line from the string.